rfcs/0001-tide-roadmap.md

```diff
@@ -0,0 +1,130 @@
+- Feature Name: N/A
+- Start Date: (fill me in with today's date, 2019-03-27)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- RustAsync Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+The roadmap for the first 6 week for the Rustasync ecosystem work group.
+
+This RFC is intended to provide the general outline and define the goals for the first sprint. The 6 week sprint provides a block time long enough to take a feature to completion, while at the same time being short enough that things do not stagnate.
+
+Since this will be the first sprint for the async ecosystems wg, the goal is to build a good foundation on which future work can proceed.
+
+* Start date: *28 March*
+* End date: *2 May*
+
+# Detailed description
+[detailed-description]: #detailed-description
+
+# Tide
+[tide]: #tide
+
+The project has seen a lot of discussion happen in [issues][issues] as well as the discord channel. The general conversation currently revolves around getting the core framework stabilized in order to build features on top of that. To this end, the currently [open PR by @aturon][context-pr] works to refactor the way data is passed along to endpoint functions. This change is a major change allowing data extraction to be more explicit.
+
+ Once this feature lands, here are some of the issues that work can begin on:
+
+ * [Authentication in Tide][issues-99]
+ * [Serving static files][issues-63]
+ * [URL generation][issues-24]
+
+There has been some discussion on the [Sprint 1 goals issue][sprint-goals], and a really good overview for the features to work on has been been [outlined by @gruberb][goals-outline]. This outline a great basis to set up the roadmap for Tide. It consists of the following broad goals:
+
+* [ ] Stabilize tide core.
+* [ ] Session management
+* [ ] Authentication
+
+## Stabilize tide core
```

Comment:
The word "stabilize" has quite some meaning. If we can follow this as more of "let's merge the first iteration of the new PR" that'd be awesome

Reply:
Yup, makes sense.